packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```
@@ -89,10 +89,12 @@
         justify-content: space-between;
         margin: 2.4em 0;
         padding: 1.6rem;
+        width: 97.5%;
 
         &--heading {
             color: var(--text-prominent);
             line-height: 1.5;
+            margin-right: 1em;
```

any reason why are we using `em` here rather than `rem`?

---

I assumed that em is better to use for margin and padding, so that the element would be more responsive as these parameters change with the font-size of the element and no need to assign new values for different screen sizes except font-size?

---

we normally use rem because it follows single source from root font size. in that case, it is easier to predict or recognize how many pixels it will hold. However, what you said is true as well, we can use both depends on the case 👍

---

sorry for the confusion, we decided to use `rem` for unit/size convention. it is better to use as per mentioned in the previous comment in addition we need to maintain code consistency.
```suggestion
            margin-right: 1.2rem;
```